userguide/storage.rst

```diff
@@ -770,6 +781,47 @@ usually not worth the performance hit.
    dataset are saved, giving the effect of deduplication without the
    overhead.
 
+#ifdef freenas
+.. _Deduplication RAM requirements:
+
+Deduplication RAM requirements
+------------------------------
+
+A generally quoted rule of thumb is 5 GB of RAM per terabyte of 
+deduplicated storage. However, if the pool is highly duplicated 
+(around 3x to 5x space saving) this can drop to as low as 1 GB to 2 
```

Comment:
The rest of these changes also have lots of asides.  Please avoid that.  Even a comma-based aside is better than parentheses, like this one, but having short, declarative sentences without pauses is best.

Reply:
updated, please re-review